app/lib/content_security_policy.rb

```diff
@@ -72,6 +72,15 @@ def s3_hostname_host
     host_to_url ENV.fetch('S3_HOSTNAME', nil)
   end
 
+  def swift_object_url
+    url = ENV.fetch('SWIFT_OBJECT_URL', nil)
+    return if url.nil?
```

Comment:
There is just one thing I'm not sure about: is `SWIFT_OBJECT_URL` necessary a URL? From the name, I assume so, but it looks like it is severely under-documented, and the underlying library we use is also unclear about that (and does not call it URL but `host`).

Maybe it is worth only running this code if we know it's a HTTPS URL?
```suggestion
    url = ENV.fetch('SWIFT_OBJECT_URL', nil)
    return if url.blank? || !url.start_with?('https://')

```

Reply:
There is not enough documents about SWIFT.
I cannot find how to setup mastodon with SWIFT compatible object storage by the host name and the directory.
So, I think the value of SWIFT_OBJECT_URL must be a URL.